service-api/module/Application/config/module.config.php

```diff
@@ -320,6 +320,16 @@
                     ],
                 ],
             ],
+            'save_address_to_case' => [
+                'type' => Segment::class,
+                'options' => [
+                    'route' => '/cases/:uuid/save-address-to-case',
+                    'defaults' => [
+                        'controller' => Controller\IdentityController::class,
+                        'action' => 'saveAddressToCase',
+                    ],
+                ],
+            ],
```

Comment:
I was just wondering regarding choosing the method and route, is there any reason we're not trying to be more restful 

So for example a restful endpoint for this would be: 
```
POST /cases/:uuid/addresses
```

Not singling this particular route out as I can see we're not following a restful approach in other places as well.

Reply:
its a good question - i'm not really sure, i guess i've just been following the pattern already in place?